package.json

```diff
@@ -27,7 +30,7 @@
   },
   "dependencies": {
     "dugite": "2.5.2",
-    "superstring": "^2.4.4",
+    "superstring": "github:pulsar-edit/superstring#de97b496663fce40050bf2d66e1466ccfbd00943",
```

Comment:
Is there any particular reason to use the commit hash instead of the release tag? I've never used the `github:` syntax for dependencies, so I'm unsure of the intricacies

Reply:
The version we’re using on `master` never had a proper release tag, even in Git. This is part of what we’ll soon address with all these version bumps and actual publishes to NPM.